install/playground/metallb.yaml

```diff
@@ -0,0 +1,13 @@
+#Replace the `addresses` field with the CIDR of Elastic IP allocated
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: my-ip-space
+      protocol: layer2
+      addresses:
+      - 147.28.141.9/32 
```

Comment:
How do we ensure that no other LoadBalancer service steals this IP from Contour?